x/finality/keeper/power_table.go

```diff
@@ -57,6 +26,14 @@ func (k Keeper) GetVotingPower(ctx context.Context, fpBTCPK []byte, height uint6
 	return sdk.BigEndianToUint64(powerBytes)
 }
 
+// GetVotingPower gets the voting power of a given finality provider at a given Babylon height
+func (k Keeper) GetVotingPower(ctx context.Context, fpBTCPK []byte, height uint64) uint64 {
+	if !k.BTCStakingKeeper.HasFinalityProvider(ctx, fpBTCPK) {
+		return 0
+	}
```

Comment:
Is this check necessary? `getVotingPower` will also return `0` if the fp does not exist in db